### ISSUE TITLE
Metabolomics Results for OSM Series 1, 3 and 4

### ISSUE THREAD

**Issue by mattodd** [OpenSourceMalaria/OSM_To_Do_List#524]
Darren Creek and his lab have carried out metabolomics work on several OSM compounds (see #469 for background). Darren constructed a powerpoint of the results (below) and provided the following commentary, which I’m posting with permission.

“We have tested those OSM compounds in our metabolomics screen and have some nice results regarding the Mode of Action of two of the actives. A full summary is attached, briefly:

OSM-S-37 strongly (and OSM-S-112 weakly) inhibit pyrimidine biosynthesis, suggesting a MoA similar to atovaquone or DSM-265. 

OSM-S-313 looks like a PfATP4 inhibitor, and matches the profiles of the three compounds MMV sent representing this class. 

OSM-S-5 and OSM-S-106 had significant effects on the parasite in our 5-hour assay, but gave a 'general death' profile which is not class-specific, so we can't pinpoint a target. We could repeat these two compounds on purified parasites to dig a bit deeper. 

OSM-S-4, OSM-S-133 and OSM-S-291 had no impact on parasite metabolism in our assay. I assume these (and OSM-S-112) were the negative controls? 

Perhaps we could arrange a TC to discuss the results and where to from here?  I am interested to follow-up 5 and 106 in more depth, but it would be good to know more context about the background and progress of these classes before investing too much into it.”

Interest in MoA for Series 3 (OSM-S-106) is described in #421.
Competition re Series 4 (OSM-S-313) is described in #503.
Publication of Series 1 (OSM-S-5, OSM-S-37) (#434) was inconclusive on MoA

This is nice, and important, work. I guess the results support the idea that Series 4 targets PfATP4. We have a possible line of enquiry for Series 1. We'll need more investigations to provide clues for Series 3. Any thoughts/suggestions below.

[Data_summary_OSM_MMV_compounds.pptx](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/1178873/Data_summary_OSM_MMV_compounds.pptx)

(This Results Issue can be closed once these data are incorporated into relevant wikis to ensure no orphaning.)

**Comment by MFernflower** [comment]
I made a quick and dirty graphic describing the data collected from the experiment: 
![e75fcbcc-c3a6-11e6-99f3-896427aabfcb](https://user-images.githubusercontent.com/3164942/28679588-a74e68fc-72c1-11e7-8dda-9f03dcf75295.png)

**Comment by MFernflower** [comment]
OSM-S-5 almost looks like a surfactant @spadavec can you model what this thing does in membrane?

**Comment by mattodd** [comment]
Thanks for the graphic @MFernflower but we lost some of the OSM codes. @alintheopen any chance you could drag the zip of the .cdx here from your original post [here](https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/469#issuecomment-267516162)? Here's the excel.
[Darren Creek Compounds 170117.xlsx](https://github.com/OpenSourceMalaria/OSM_To_Do_List/files/1182299/Darren.Creek.Compounds.170117.xlsx)

**Comment by MFernflower** [comment]
@mattodd updated graphic: 
![e75fcbcc-c3a6-11e6-99f3-896427aabfcb](https://user-images.githubusercontent.com/3164942/28716115-5e0b7a24-7369-11e7-9ad0-2534c260748c.png)

**Comment by spadavec** [comment]
@MFernflower Not really--it would be _very_ exploratory MD, which is something that I don't really feel comfortable with. There might be a fun way to calculate dG_insertion into a membrane for that compound (assuming its passive transport), but not sure how informative that would be.

**Comment by MFernflower** [comment]
A possible way to test my surfactant hypothesis would be to make analouges of OSM-S-5 with various hydrophobic and hydrophilic groups replacing the F atom

**Comment by drc007** [comment]
@MFernflower Out of curiosity which surfactant do you feel they look similar to? Most I know of have an ionisable group.

**Comment by MFernflower** [comment]
I would assume the free N atom on the tail would ionize while the front of
the molecule acts as a hydrophobic head

**Comment by mattodd** [comment]
I bumped into Darren, Anubhav and Stuart Ralph in Melbourne last week. We talked about these results. Points to note:

1) Series 1 (OSM-S-5) and 1b (OSM-S-112) gave different profiles. This does agree with Corey Nislow’s original analysis in the [Series 1 paper](http://pubs.acs.org/doi/full/10.1021/acscentsci.6b00086). Interestingly there is the suggestion that the MoA is related to DHODH inhibition, but this is possible in two ways - direct inhibition of the enzyme, or [inhibition of the associated electron transport events](http://www.cell.com/action/showImagesData?pii=S1471-4922%2816%2930170-2) (not DHODH itself) [which is how atovaquone works](http://www.sciencedirect.com/science/article/pii/S1383576914001330). As part of the Series 1 paper we evaluated the compounds vs DHODH, which clearly showed no inhibition. We could see whether the MoA is similar to atovaquone since Stuart Ralph has an assay in which plasmodium DHODH is expressed in yeast (I think) which lacks the electron recycling. If the compounds are no longer potent in this model, the compounds have the atovaquone MoA. Stuart is offering to run this, maybe in a month or so. I will keep in touch with him re this, unless anyone has any contrary suggestions.

2) The result for OSM-S-106 is unclear. The modeling by Vito (#503) suggests Sir2a as a target. Stuart is also offering to look at this, perhaps with Chris Tonkin. They have single knockouts of both Sir2’s. Interestingly I hear there is a double-knockout of Sir2 which is viable in Berghei, i.e. it would seem that sir2 is not a drug target (again, for Berghei). Nevertheless, we should investigate.

3) It’s on me to talk to Christian Doerig to see if we can arrange a clk assay, again to validate Vito’s predictions in #503, again because the metabolomics results were not conclusive for OSM-S-106.

(3a - Separately we have received approval to send some OSM-S-106 to Elizabeth Winzeler's lab for MoA analysis, presumably by the creation of resistant mutants. More on this when we send compound and know about timelines - it's an in-kind support from MMV in response to their recent call for compounds to be investigated further in this way. Communicated to me by email a couple of weeks ago).

4) The profile for the Series 4 compounds seems quite cleanly in favor of PfATP4, though the metabolomic profile is actually quite similar to the “general death” seen for other compounds. Again, the results do not imply that PfATP4 is the molecular target, only that Series 4 shares a MoA with e.g. the spiroindolones. But the MoA could still be something else.

**Comment by MFernflower** [comment]
Now that you say it - OSM-S-112 does indeed look like it has potential to be a redox cycler - I wonder how that would be tested in a lab however - some sort of eletrochemical cell? @mattodd

**Comment by holeung** [comment]
Is there interest in following up on OSM-S-37?

**Comment by MFernflower** [comment]
I'd be curious if you could dock it to DHODH so we could have some sort of model? @holeung

**Comment by mbhebhe** [comment]
OSM-S-106 was shipped to UCSD last week for Mode of Action studies
![image](https://user-images.githubusercontent.com/26862286/32202684-73aa0274-be32-11e7-8c0e-699f86c5f2e7.png)

MMV # | OSM # | SMILES | InChl | InChl Key
-- | -- | -- | -- | --
MMV025100 | OSM-S-106 | NC1=C2C(C=C(S2)C3=CC(S(=O)(N)=O)=CC=C3)=NC=N1 | InChI=1S/C12H10N4O2S2/c13-12-11-9(15-6-16-12)5-10(19-11)7-2-1-3-8(4-7)20(14,17)18/h1-6H,(H2,13,15,16)(H2,14,17,18) | MQMXDJVOZKMSNT-UHFFFAOYSA-N

**Comment by mattodd** [comment]
Just FYI there was a mixup of codes used in the above (re OSM-S-313 and OSM-S-291) - see OpenSourceMalaria/OSMSeries4Paper1#36
